main.go

```diff
@@ -377,17 +379,18 @@ func main() {
 }
 
 type configuration struct {
-	Verbosity   int
-	Debug       bool
-	DebugListen string
-	API         api.Config
-	Database    db.Config
-	Temporal    temporal.Config
-	Watcher     watcher.Config
-	Pipeline    []pipeline.Config
-	Validation  validation.Config
-	Telemetry   TelemetryConfig
-	Metadata    metadata.Config
+	Verbosity      int
+	Debug          bool
+	DebugListen    string
+	API            api.Config
+	ExtractArchive archive.Config
```

Comment:
Based on the config changes [here](https://github.com/artefactual-labs/enduro/pull/605/files#diff-cc2139700f8207bdcf502d8c0c22b6f47567663a6f75468cccede2f88310114cR24) this should be `ExtractActivity`.

Reply:
Hehe, oops. Good catch @jraddaoui. :)